.laminas-ci.json

```diff
@@ -1,6 +1,6 @@
 {
     "ignore_php_platform_requirements": {
-        "8.4": true
+        "8.4": false
```

Comment:
this can be removed entirely then?

Reply:
Yes, but non-blocker

Reply:
I leave it there for 8.5 or 9.0 so you don't have to goo looking for the correct variable name next time